go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/grafana/grafana
 
 go 1.23.5
 
+replace github.com/grafana/authlib => /Users/charandas/go/src/github.com/grafana/authlib
```

Comment:
@charandas looks like a leftover from testing?

Reply:
Yes, its a replace from my branch pending in authlib. Once that is merged, I will remove it.

Reply:
Got it, thanks!